docsource/modules150-160.rst

```diff
@@ -110,7 +110,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | crm_livechat                                    | Nothing to do        |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| crm_mail_plugin                                 |                      |                                                 |
+| crm_mail_plugin                                 | Nothing to do        |                                                 |
```

Comment:
@NgoHoangHung Change is happening in PR https://github.com/OCA/OpenUpgrade/pull/4063